modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/snapshot/programs/SnapshotProcessor.scala

```diff
@@ -236,6 +239,13 @@ sealed abstract class SnapshotProcessor[F[_]: Async: KryoSerializer: SecurityPro
               obsoleteToRemove
             )
           }
+
+      case Ignore(lastHeight, lastSubHeight, lastOrdinal, processingHeight, processingSubHeight, processingOrdinal) =>
+        logger
+          .warn(
+            s"Unexpected case during global snapshot processing - ignoring snapshot! Last: (height: $lastHeight, subHeight: $lastSubHeight, ordinal: $lastOrdinal) processing: (height: $processingHeight, subHeight:$processingSubHeight, ordinal: $processingOrdinal)."
```

Comment:
We may also log the expected and actual lastSnapshotHash as these may also differ. But let's add it in another PR.